src/main/java/org/spongepowered/common/mixin/api/mcp/world/ExplosionMixin_API.java

```diff
@@ -110,4 +111,9 @@ public double getKnockback() {
         return  ((ExplosionBridge) this).bridge$getKnockback();
     }
 
+    @Override
+    public Optional<ResistanceCalculator> getResistanceCalculator() {
```

Comment:
Given that there is always a `ResistanceCalculator`, this shouldn't be optional.

Reply:
I believe there are other projects that use the SpongeAPI as a base or so I was told with my last PR.